.github/workflows/test_inf2.yml

```diff
@@ -23,6 +23,16 @@ jobs:
     env:
       AWS_REGION: us-east-1
     steps:
+      - name: Install Neuron runtime
+        run: |
+          . /etc/os-release
+          sudo tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
+          deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
+          EOF
+          wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | sudo apt-key add -
+          sudo apt-get update -y
+          sudo apt-get install aws-neuronx-tools aws-neuronx-oci-hook aws-neuronx-runtime-lib aws-neuronx-collectives -y
+          export PATH=/opt/aws/neuron/bin:$PATH
       - name: Check AMI
```

Comment:
You can remove the step where I displayed the AMI package versions then.
Also, we will always get the latest version of the packages:
- on one hand, this is good because we won't have to update the builders when a new SDK is released,
- on another hand it might break if the python packages are not compatible.

I think it is worth the risk, but would like to head what @JingyaHuang @michaelbenayoun think.

Reply:
We can install specific version if you prefer . as you want

Reply:
After discussion, since the dkms comes preinstalled in the builder with a pinned version, then yes.